Example/Podfile

```diff
@@ -4,3 +4,11 @@ use_frameworks!
 target 'ReusableDemo' do
   pod 'Reusable', :path => '../'
 end
+
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings['SWIFT_VERSION'] = '3.0'
+    end
+  end
+end
```

Comment:
This could be done in shorter way:
```
post_install do |installer|
    installer.pods_project.build_configurations.each do |config|
        config.build_settings['SWIFT_VERSION'] = '3.0'
    end
end
```